src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/GerritTrigger.java

```diff
@@ -133,7 +133,7 @@ public class GerritTrigger extends Trigger<Job> {
     private Integer gerritBuildAbortedVerifiedValue;
     private Integer gerritBuildAbortedCodeReviewValue;
     private boolean silentMode;
-    private boolean enableTopicAssociation = Config.DEFAULT_ENABLE_TOPIC_ASSOCIATION;
```

Comment:
Question from my side, can such a change break the job options if the update is applied?

Reply:
Yes it will fail to be read from disk. You need to mark the old field as `transient` and then in `readResolve` if the field is set migrate it to the new field.
See https://www.jenkins.io/doc/developer/persistence/backward-compatibility/